### PR TITLE
website: Adding slides from previous workshops

### PR DESCRIPTION
**Pull request by ivanaamit**

### CODE REVIEW

**Comment by ivanaamit** [comment]
@mattsinc, please let me know if you'd like me to make any changes. For example, if you prefer a different name for the slides instead of "GPU", just let me know. Thanks.

File: _pages/community/events/hpca-2023.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ permalink: events/hpca-2023
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/npbMXXoKltg" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
-Slides: [PDF](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pdf), [Powerpoint](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pptx).
+Slides: 
+- [PDF](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pdf), [Powerpoint](/assets/files/hpca2023-tutorial/gem5-tutorial-hpca-2023.pptx).
```
Comment (mattsinc) [inline _pages/community/events/hpca-2023.md]: Maybe this is a dumb question @ivanaamit , but are the slides for the non-GPU sections not posted?  I don't see them anywhere on this md page -- maybe they are all in the codespace?

Separately, I think we can just post the PDF, no need for PPTX too.

For name, I think "GPU" is fine!

Comment (ivanaamit) [reply _pages/community/events/hpca-2023.md]: I fixed it, the previous "PDF" link was for the non-GPU slides.

File: _pages/community/events/isca-2024.md [hunk 2]
```diff
@@ -8,10 +8,11 @@ The 7th gem5 tutorial will be held in conjunction with the workshop.
...
 ### Slides
-- [intro](/assets/files/isca2024-tutorial/01-intro.pdf)
-- [getting started](/assets/files/isca2024-tutorial/02-getting-started.pdf)
-- [standard library](/assets/files/isca2024-tutorial/03-standard-library.pdf)
-- [sim object](/assets/files/isca2024-tutorial/04-sim-object.pdf)
+- [Intro](/assets/files/isca2024-tutorial/01-intro.pdf)
+- [Getting started](/assets/files/isca2024-tutorial/02-getting-started.pdf)
+- [Standard library](/assets/files/isca2024-tutorial/03-standard-library.pdf)
+- [SimObject](/assets/files/isca2024-tutorial/04-sim-object.pdf)
+- [GPU](/assets/files/isca2024-tutorial/05-gpu.pdf)
```
Comment (ivanaamit) [inline _pages/community/events/isca-2024.md:15]: @mattsinc, is the "GPU" title appropriate for the ISCA slides?

**Review by mattsinc — APPROVED** [review]

**Pull request merged**